CRM/Logging/Schema.php

```diff
@@ -302,7 +302,7 @@ public function fixSchemaDifferences($enableLogging = FALSE) {
    * and also implements the engine change defined by the hook (i.e. INNODB).
    *
    * Note changing engine & adding hook-defined indexes, but not changing back
-   * to ARCHIVE if engine has not been deliberately set (by hook) and not dropping
+   * to INNODB if engine has not been deliberately set (by hook) and not dropping
    * indexes. Sysadmin will need to manually intervene to revert to defaults.
    */
   public function updateLogTableSchema() {
```

Comment:
Can I suggest that here and in the naming of the API call `System.updatelogtables`, we are explicit about what this actually does? Eg `convertDataLoggingTablesFromArchiveToInnoDB()` is explicit, where "updateLogTables" could mean many things.

* In future there will be other updates, and a function named like this will occupy a namingspace that also fits other functions.
* Who wouldn't run a function called "update"? Sounds reasonable, but [as @Stoob points out](https://lab.civicrm.org/dev/core/issues/663#note_17556), this process has an impact on a functioning site.
* CiviCRM has a pattern of recycling terminology in multiple spaces (I count [six different meanings for "log"](https://gist.github.com/xurizaemon/01a53b0e63b73f88de61dca5d9afe80a)), I think it's helpful for both DX and UX if we refer to things using clear terminology.

Reply:
Comment above was written thinking you'd added the function in this PR - on second view this need not block your proposed changes (but please consider it where adding new methods etc).

Reply:
I agree that the current naming is confusing - I've spent time in this code in the past, but I still get `System.updatelogtables` mixed up with `System.createmissinglogtables` all the time.

What makes it a bit tricky is that, in addition to changing the engine schema, the function/API will potentially also do the following things:

- Change the column definition of `log_conn_id`
- Create any missing indexes (which may have been added via `hook_civicrm_alterLogTables`)

I'm okay with renaming the existing API if we can come up with a less confusing name for the API that encapsulates all of that.